audit-opentelemetry/chart/alerts/collector-alerts.yaml

```diff
@@ -0,0 +1,41 @@
+groups:
+  - name: collector-alerts
+    rules:
+  {{- if not (has "FilelogRefusedLogs" .Values.openTelemetry.prometheus.rules.disabled) }}
+      - alert: FilelogRefusedLogs
+        expr: sum(rate(otelcol_receiver_refused_log_records_total{receiver=~"filelog"}[1m])) > 0
+        for: 5m
+        labels:
+          severity: warning
+          runbook_url: https://github.com/cloudoperators/greenhouse-extensions/tree/main/opentelemetry/playbooks/FilelogRefusedLogs.md
```

Comment:
even if its just a boilerplate for now, you can change it to referring to audit-opentelemetry, same for the other alerts here and in operator-alerts.yaml